src/ConnectedMode.UnitTests/Migration/FileProviders/XmlFileProviderTests.cs

```diff
@@ -118,45 +150,59 @@ public async Task GetFiles_HasSolutionFolder_ExpectedFilesReturned()
             var projectProvider = CreateRoslynProjectProvider("project1", "project2");
 
             var filesToReturn = new string[] { "file1", "file2", "file3" };
-            var fileSystem = CreateFileFinder(filesToReturn);
+            var fileFinder = CreateFileFinder(filesToReturn);
 
-            var testSubject = CreateTestSubject(serviceProvider.Object, fileSystem.Object, projectProvider.Object);
+            var testSubject = CreateTestSubject(serviceProvider.Object, fileFinder.Object, projectProvider.Object);
 
             var actual = await testSubject.GetFilesAsync(CancellationToken.None);
 
             // Expecting the project files and any files found by searching
             actual.Should().BeEquivalentTo("file1", "file2", "file3", "project1", "project2");
             solution.VerifyAll();
 
-            fileSystem.Verify(x => x.GetFiles("root dir", SearchOption.AllDirectories, XmlFileProvider.FileSearchPatterns), Times.Once);
+            fileFinder.Verify(x => x.GetFiles("root dir", SearchOption.AllDirectories, XmlFileProvider.FileSearchPatterns), Times.Once);
         }
 
         [TestMethod]
         public async Task GetFiles_DuplicatesAreIgnored()
         {
-            var solution = CreateIVsSolution("root dir");
+            var solution = CreateIVsSolution("solution dir");
             var serviceProvider = CreateServiceProviderWithSolution(solution.Object);
-            var projectProvider = CreateRoslynProjectProvider("the project file");
+            var projectProvider = CreateRoslynProjectProvider("c:\\projectFolder\\the project file");
```

Comment:
Checks that the results of the project folder search are included in the de-duplication.